config.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+type AgentOidcConfig struct {
+	ClientID string `json:"clientID"`
+	AuthUrl  string `json:"authUrl"`
+	TokenUrl string `json:"tokenUrl"`
+}
+
+type AgentPeerConfig struct {
+	Url string `json:"url"`
+}
+
+type AgentDevConfig struct {
+	Name  string             `json:"name"`
+	Peers *[]AgentPeerConfig `json:"peers"`
+}
+
+type AgentConfig struct {
+	Oidc *AgentOidcConfig  `json:"oidc"`
+	Devs *[]AgentDevConfig `json:"devs"`
+}
+
+func UnmarshallAgentConfig(data []byte, v interface{}) error {
```

Comment:
nitpick: do we really need this alias?
Not really opposed to it, it just feels more natural to use `json.Unmarshal` directly.
Also Unmarshal is with a single l )

Reply:
tbh, that alias was handy for me to test reading the config while writing this file. Fixing the typo